components/eam/bld/configure

```diff
@@ -1387,9 +1387,27 @@ if (($chem_pkg ne 'none') || ($prog_species)) {
 if ($chem_pkg =~ '_mam3') {
     $chem_cppdefs = ' -DMODAL_AERO -DMODAL_AERO_3MODE ';
 } elsif ($chem_pkg =~ '_mam4_mom' || $chem_pkg =~ '_mam4_resus_mom' ) {
+#for turning on CLDERA stratospheric volcanic aerosol treatment
+#    $chem_cppdefs = ' -DMODAL_AERO -DMODAL_AERO_4MODE_MOM ';
+    if (!defined $opts{'cldera_strat_volc'} ) {
     $chem_cppdefs = ' -DMODAL_AERO -DMODAL_AERO_4MODE_MOM ';
+    }
+    else{
```

Comment:
As a matter of style/formatting, I'd prefer to see the `else on the same line as the closing brace of the `if`. I.e.,:
```
if (!defined $opts{'cldera_strat_volc'}) {
    $chem_cppdefs = ' -DMODAL_AERO -DMODAL_AERO_4MODE_MOM ';
} else {
    ...
}
```
Also, you should line up your comments with the block of code they refer to, and the code inside the else block should be indented.